aws/container-linux/kubernetes/security.tf

```diff
@@ -14,11 +14,21 @@ resource "aws_security_group" "controller" {
 resource "aws_security_group_rule" "controller-ssh" {
   security_group_id = "${aws_security_group.controller.id}"
 
-  type        = "ingress"
-  protocol    = "tcp"
-  from_port   = 22
-  to_port     = 22
-  cidr_blocks = ["0.0.0.0/0"]
+  type                     = "ingress"
+  protocol                 = "tcp"
+  from_port                = 22
+  to_port                  = 22
+  source_security_group_id = "${aws_security_group.worker.id}"
```

Comment:
This ensures that connections initiated via Tunnelbox or any other pod can reach port 22 👍